plugins/vault-backend/src/config/config.ts

```diff
@@ -27,6 +27,11 @@ export interface VaultConfig {
    */
   baseUrl: string;
 
+  /**
+   * The publicUrl for your Vault instance (Optional).
+   */
+  publicUrl?: string;
```

Comment:
You need to update getVaultConfig to actually read this out of the config as well right?

Reply:
Good catch! Fixed